code/game/area/areas/ruins/blackmarketpackers.dm

```diff
@@ -1,26 +1,23 @@
 //Packer Ship Areas
 
-/area/awaymission/BMPship
+/area/ruin/unpowered/BMPship
 	name = "\improper BMP Asteroids"
 	icon_state = "away"
```

Comment:
might want to change the icon state of these.

Reply:
Some of the ruins use the `away` icon and I plan to convert them all to just `ruin1` `ruin2` when we fully nuke the gateway. Can add those states now if you want.

Reply:
preferably you would add them now.

Reply:
Will do

Reply:
![image](https://user-images.githubusercontent.com/25063394/181909584-c1ada33d-539c-4d5b-8dd4-6588edf0dfa2.png)